main.go

```diff
@@ -157,3 +158,23 @@ func registerCapabilities(port string, log logger.Handler) {
 		log.Info(err.Error())
 	}
 }
+func registerDynamicCapabilities(port string, log logger.Handler) {
```

Comment:
Is this not a function that we're defining in the Adapter Library?

Reply:
no

Reply:
Ok... and why not?

Reply:
registerCapabilities function is also not defined in adapter library and each adapter has that function. So I did the same with this function too as I thought the function is small enough to not go in the adapter library.

Reply:
I hear that - about it being small. It's not obvious that this (your) approach is "wrong". 

It's just that we're repeating ourselves, repeating ourselves, repeating ourselves...

@mgfeller I wonder if you have a perspective on where to draw the line on items like these.